crates/numeric_cast/src/truncating.rs

```diff
@@ -51,6 +51,18 @@ truncating_cast!(u128 => u16 );
 truncating_cast!(u64  => u32 );
 truncating_cast!(u128 => u32 );
 truncating_cast!(u128 => u64 );
+
+truncating_cast!(usize => u8    );
+truncating_cast!(usize => u16   );
+truncating_cast!(usize => u32   );
+truncating_cast!(usize => u64   );
+truncating_cast!(usize => u128  );
```

Comment:
`usize/isize => u128/i128` is actually extending cast since we are unlikely to have a 128-bit computer.